<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AeroChinMain</class>
 <widget class="QMainWindow" name="AeroChinMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AeroChinquihue - Principal</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(202, 202, 202);
border-top-color: rgb(98, 98, 98);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>Bienvenido a AeroChinquihue</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>391</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>En la siguiente tabla se muestran los precios por vuelos</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>391</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string> y por encomienda a cada ciudad</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>140</y>
      <width>461</width>
      <height>321</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../tabla_ciudades.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnVuelos">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>250</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Vuelos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEncomiendas">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Encomiendas</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>756</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMi_cuenta">
    <property name="title">
     <string>Mi cuenta</string>
    </property>
    <addaction name="btnVerMiCuenta"/>
   </widget>
   <addaction name="menuMi_cuenta"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionVer_Cuenta">
   <property name="text">
    <string>Ver Cuenta</string>
   </property>
  </action>
  <action name="btnVerMiCuenta">
   <property name="text">
    <string>Ver mi cuenta</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
